### PR TITLE
LOLCODE

### PR DESCRIPTION
**Pull request by ducky64**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ducky64/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: LOL.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# LOL!!!!
+[ROTFLOL]()
```
Comment (ducky64) [inline LOL.md]: lolwut

Comment (ducky64) [inline LOL.md]: !fix

File: LOL.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# LOL!!!!
```
Comment (ducky64) [inline LOL.md]: is broken !resolve

File: LOL.md (hunk 1 above)
Comment (ducky64) [inline LOL.md]: !fixed

File: LOL.md (hunk 2 above)
Comment (ducky64) [inline LOL.md]: !resolved

**Pull request merged**